cliquet/permission/redis.py

```diff
@@ -79,6 +79,23 @@ def object_permission_principals(self, object_id, permission):
         return self._decode_set(members)
 
     @wrap_redis_error
+    def principals_accessible_objects(self, principals, permission,
+                                      object_id_match=None):
+        if object_id_match is None:
+            object_id_match = '*'
+        key_pattern = 'permission:%s:%s' % (object_id_match, permission)
+
+        # XXX: not optimal at all.
+        objects = set([])
+        _, keys = self._client.scan(match=key_pattern)
```

Comment:
This doesn't work because scan uses pagination and you don't navigate through pages here.